core/templates/dev/head/pages/landing/fractions/landing_page_teacher.html

```diff
@@ -382,30 +386,84 @@ <h2 class="oppia-fractions-landing-h2 oppia-fractions-landing-centered-h2" style
       .oppia-fractions-landing-learn-more,
       .oppia-fractions-landing-get-started,
       .oppia-fractions-landing-explore-lessons {
-        font-size: 2vw;
+        font-size: 2.8vw;
         margin-bottom: 30px;
         width: auto;
       }
       .oppia-fractions-landing-get-started {
         margin-right: 5%;
       }
       .oppia-fractions-landing-get-started-mobile {
-        left:15px;
+        left: 15px;
       }
       .oppia-fractions-landing-explore-lessons-mobile {
-        left:6.5px;
+        left: 6.5px;
       }
       .oppia-fractions-landing-image-mobile {
         display: block;
+        left: 0;
+      }
+      .oppia-fractions-landing-image-desktop {
+        display: none;
+      }
+      .oppia-fractions-landing-centered-h2 {
+        margin-left: 10%;
+        margin-right: 10%;
+        left: 0;
+      }
+    }
+    @media screen and (max-width: 450px) {
+      .oppia-fractions-landing-section-inner,
+      .oppia-fractions-landing-section {
+        height: auto;
+      }
+      .oppia-fractions-landing-h1 {
+        font-size: 1.6em;
         margin-left: 10%;
+        margin-right: 20%;
+        padding-left: 0 !important;
```

Comment:
Here and elsewhere: we generally avoid using !important in CSS rules, since it overrides the natural selection order. Could you please find a way to achieve the desired visual effect, but without using !important?

See CSS style guide [here](https://github.com/oppia/oppia/wiki/Coding-style-guide#css).

Reply:
Oh definitely, I think I can achieve the desired visual effect without the extra media query.